core/classes/Core/Util.php

```diff
@@ -128,6 +129,80 @@ public static function isExternalURL(string $url): bool {
         return !(str_replace('www.', '', rtrim(self::getSelfURL(false), '/')) == str_replace('www.', '', $parsed['host']));
     }
 
+    /**
+     * Checks whether the client making the request is a trusted proxy. If not,
+     * abruptly aborts the request using die().
+     */
+    private static function ensureTrustedProxy(): void {
+        $trustedProxies = Config::get('core/trustedProxies');
```

Comment:
Are we leaving this new `trustedProxies` configuration setting undocumented? Would make sense since it is for more advanced setups to not include it by default in the config...

Reply:
Good point, while it might sound advanced I think potentially many users need this option, especially with the prevalence of CloudFlare. Maybe it could be part of the installer.

Reply:
Documented now: https://docs.namelessmc.com/en/trusted-proxies